src/data/historikk/types/historikkTypes.ts

```diff
@@ -6,6 +6,7 @@ export type HistorikkEventType =
   | "ARBEIDSUFORHET"
   | "MANGLENDE_MEDVIRKNING"
   | "FRISKMELDING_TIL_ARBEIDSFORMIDLING"
+  | "VEILEDER"
```

Comment:
Tror denne kan oppleves tvetydig. Kunne dette vært mer presist? 😊 Eller noe lignende?
```suggestion
  | "VEILEDER_TILDELING"
```